lumigator/python/mzai/backend/backend/config_templates.py

```diff
@@ -61,15 +62,60 @@
     }}
 }}"""
 
-config_template = {
-    "hf://facebook/bart-large-cnn": bart_template,
-    "hf://mikeadimech/longformer-qmsum-meeting-summarization": seq2seq_template,
-    "hf://mrm8488/t5-base-finetuned-summarize-news": seq2seq_template,
-    "hf://Falconsai/text_summarization": seq2seq_template,
-    "hf://mistralai/Mistral-7B-Instruct-v0.3": causal_template,
-    "oai://gpt-4o-mini": oai_template,
-    "oai://gpt-4-turbo": oai_template,
-    "oai://gpt-3.5-turbo-0125": oai_template,
-    "mistral://open-mistral-7b": oai_template,
-    "llamafile://mistralai/Mistral-7B-Instruct-v0.2": oai_template,
+# Inference templates
+
+seq2seq_infer_template = """{{
+    "name": "{job_name}/{job_id}",
+    "model": {{ "path": "{model_path}" }},
+    "dataset": {{ "path": "{dataset_path}" }},
+}}"""
+
+bart_infer_template = """{{
+    "name": "{job_name}/{job_id}",
+    "model": {{ "path": "{model_path}" }},
+    "tokenizer": {{ "path": "{model_path}", "mod_max_length": 1024 }},
+    "dataset": {{ "path": "{dataset_path}" }},
+}}"""
+
+causal_infer_template = """{{
+    "name": "{job_name}/{job_id}",
+    "model": {{ "path": "{model_path}" }},
+    "dataset": {{ "path": "{dataset_path}" }},
+}}"""
+
+oai_infer_template = """{{
+    "name": "{job_name}/{job_id}",
+    "model": {{
+        "inference": {{
+            "base_url": "{model_url}",
+            "engine": "{model_path}",
+            "system_prompt": "{system_prompt}",
+            "max_retries": 3
+        }}
+    }},
+    "dataset": {{ "path": "{dataset_path}" }},
+}}"""
+
+config_infer_template = {
+    "hf://facebook/bart-large-cnn": bart_infer_template,
```

Comment:
I made the inference set of models less than the ones available for evaluation per our previous conversations that ground truth should offer a small set of models, let me know what you think @aittalam

Reply:
I agree on GT offering a smaller set of models but I would not constrain it here, the main reason being batch inference as a first step for evaluation (so we need to support all the models we currently support in eval).
We can think about this list as "all the models we want to make available for batch inference", and then constrain which ones we suggest for GT via the API (i.e. with a `(task, model_list)` mapping, similarly to what we want to do for eval itself). WDYT?

Reply:
Yup, that sounds good, left this list in-tact for now